src/pudl/settings.py

```diff
@@ -476,6 +483,23 @@ class Ferc2XbrlToSqliteSettings(FercGenericXbrlToSqliteSettings):
     taxonomy: AnyHttpUrl = "https://eCollection.ferc.gov/taxonomy/form2/2022-01-01/form/form2/form-2_2022-01-01.xsd"
 
 
+class Ferc2DbfToSqliteSettings(GenericDatasetSettings):
```

Comment:
It seems like we should have a validator in this and the other DBF and XBRL to SQLite settings classes that behaves like the one in `Ferc1Settings` such that if someone requests an invalid year of data we can either fix it or raise an error.